### DISCUSSION TITLE
Shutdown box

### DISCUSSION THREAD

**Discussion by a-dasler** [devmfc/debian-on-amlogic#7] [Q&A]
Hello, if it is possible to correctly shutdown the tvbox with a button? Editing logind.conf has no result.

**Comment by devmfc** [comment]
I don't know. My S905X box always keeps rebooting, not shutting down. I think you see the same now. It guess it has something to do with the combination of bootloader and kernel.

**Comment by a-dasler** [comment]
Is the button rebooting? Mine just doesn't respond. Well, okay, not critical, thanks for the answer!
